backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/codec/impl/AbstractLuceneNumericFieldCodec.java

```diff
@@ -85,15 +85,15 @@ public boolean isCompatibleWith(LuceneFieldCodec<?> obj) {
 
 		AbstractLuceneNumericFieldCodec<?, ?> other = (AbstractLuceneNumericFieldCodec<?, ?>) obj;
 
-		return ( projectable == other.projectable ) && ( searchable == other.searchable )
-				&& ( sortable == other.sortable ) && ( aggregable == other.aggregable );
+		return (projectable == other.projectable) && (searchable == other.searchable)
+			&& (sortable == other.sortable) && (aggregable == other.aggregable);
```

Comment:
This probably won't pass the style checks.

Reply:
I compiled with control. I always have style control turned on. Passed correctly. But it always adjusts the autoformat parameters. Unfortunately, there is no setting for this style.